PRs failing due to problem with System.Collections.Immutable package

This is taking out a bunch of PRs this afternoon / evening:
```
Discovering: System.Collections.Immutable.Tests
  System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
d:\j\workspace\dotnet_corefx_windows_debug_prtest\packages\Microsoft.DotNet.BuildTools.1.0.25-prerelease-00045\lib\tests.targets(101,5): warning MSB3073: The command "corerun.exe xunit.console.netcore.exe System.Collections.Immutable.Tests.dll  -xml testResults.xml -notrait category=nonwindowstests  -notrait category=OuterLoop -notrait category=failing " exited with code 1. [d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\System.Collections.Immutable\tests\System.Collections.Immutable.Tests.csproj]
d:\j\workspace\dotnet_corefx_windows_debug_prtest\packages\Microsoft.DotNet.BuildTools.1.0.25-prerelease-00045\lib\tests.targets(112,5): error : One or more tests failed while running tests from 'System.Collections.Immutable.Tests' please check log for details! [d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\System.Collections.Immutable\tests\System.Collections.Immutable.Tests.csproj]
C:\Program Files (x86)\MSBuild\12.0\bin\Microsoft.Common.CurrentVersion.targets(1697,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. 
```

--- a/src/System.Collections.Immutable/tests/System.Collections.Immutable.Tests.csproj
+++ b/src/System.Collections.Immutable/tests/System.Collections.Immutable.Tests.csproj
@@ -74,6 +74,7 @@
     <ProjectReference Include="..\src\System.Collections.Immutable.csproj">
       <Project>{1dd0ff15-6234-4bd6-850a-317f05479554}</Project>
       <Name>System.Collections.Immutable</Name>
+      <Private>true</Private>
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>

